web/index.php

```diff
@@ -175,8 +175,8 @@
 isset($request) || $request = NULL;
 isset($action) || $action = NULL;
 
-if ( ZM_ENABLE_CSRF_MAGIC && $action != 'login' ) {
-    Debug("Calling csrf_check with the following values: \$request = \"$request\", \$view = \"$view\", \$action = \"$action\"");
+if ( ZM_ENABLE_CSRF_MAGIC ) {
```

Comment:
Right, I took it out since it was only a workaround for not having a proper way to authenticate to the API which this bug is resolving.